code/modules/cargo/packs.dm

```diff
@@ -1536,3 +1536,17 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
 	crate_name = "Cigarettes Cartons Crate"
 	containertype = /obj/structure/closet/crate
 	group = "Supply"
+
+/datum/supply_pack/randomised/corporate
+	num_contained = 3
+	contains = list(/obj/item/clothing/suit/space/void/merc,
```

Comment:
One of the best voidsuits? And antag only crossbows?

Reply:
That's why it costs 8k and is hidden behind contraband

Reply:
> Just another buff to guild, that actually should be nerfed.
> Plus "contraband" doesn't make sense, since there is nothing illegal that guild can buy or sell.

You need to set the cargo desk board to broad with a multitool, which not a lot of people know you can do, plus it's one of the most expensive things guild can buy if not the most, remember that guild can also get the crimson RIG from surplus rig crates which is extremely OP, and that the contents are randomised, you could get those shitty pistols for 8k, and the crossbows deal only 25 and 30 toxin respectively

Reply:
The *chance* to roll a crimson RIG costs 5k on itself, and IMO a blood red is way better due to not taking up your back slot

Reply:
> The _chance_ to roll a crimson RIG costs 5k on itself, and IMO a blood red is way better due to not taking up your back slot

Here you have a chance too, you can either roll shitty pistols, or the blood red, and for 8k, I can increase the price even more if that's still not balanced

Reply:
And like jooks said below guild can buy the best ballistic armor with nightvisions which contains 2 sets for just 3k